### PR TITLE
Update from Upstream 1

### PR DESCRIPTION
**Pull request by Razgriz1032**
🔥🔥🔥🔥🔥🔥🔥🔥

### CODE REVIEW

**Comment by Razgriz1032** [comment]
In hindsight I probably should have made the resleeving sickness a separate PR. Butt fuck it.

**Comment by Razgriz1032** [comment]
Finally

File: code/datums/autolathe/engineering.dm [hunk 1]
```diff
@@ -102,3 +102,10 @@
 /datum/category_item/autolathe/engineering/camera_assembly
 	name = "camera assembly"
 	path =/obj/item/weapon/camera_assembly
+
+//YW Addition Start
+/datum/category_item/autolathe/engineering/rpd
+	name = "rapid piping device"
+	path =/obj/item/weapon/pipe_dispenser
+//YW Addtion End	
+
```
Comment (TheFurryFeline) [inline code/datums/autolathe/engineering.dm]: Probably should make a _yw.dm file for these.

File: code/__defines/construction.dm [hunk 2]
```diff
@@ -55,7 +55,29 @@
...
+#define DISPOSAL_PIPE_SORTER_FLIPPED 10
+#define DISPOSAL_PIPE_UPWARD 11
+#define DISPOSAL_PIPE_DOWNWARD 12
+#define DISPOSAL_PIPE_TAGGER 13
+#define DISPOSAL_PIPE_TAGGER_PARTIAL 14
 
+#define DISPOSAL_SORT_NORMAL 0
+#define DISPOSAL_SORT_WILDCARD 1
+#define DISPOSAL_SORT_UNTAGGED 2
+//YW Additions end
```
Comment (TheFurryFeline) [inline code/__defines/construction.dm]: _yw.dm file addition to be done.

**Pull request merged**